Services captiveportal filemanager

sbeaver-netgate opened the pull request:
Ready for review

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
services_captieveportal_filemanager.php Conversion complete

Ready for review

sbeaver-netgate committed:
remove unneeded gettext/htmlspecialcharacters

SjonHortensius commented on usr/local/www/services_captiveportal_filemanager.php:
```diff
@@ -135,100 +137,162 @@ function cpelements_sort() {
-</body>
-</html>
+	"file manager. Then you can include it in your portal page like this:")?><br /><br />
+	<tt>&lt;img src=&quot;captiveportal-test.jpg&quot; width=... height=...&gt;</tt><br /><br />
```
`<tt>` is no longer supported in html5, I think you should use `<section>` for this http://getbootstrap.com/css/#code

SjonHortensius commented on usr/local/www/services_captiveportal_filemanager.php:
```diff
@@ -135,100 +137,162 @@ function cpelements_sort() {
+	</div>
+</div>
+
+<script>
```
This script is correctly written, but why would this page need toggleable notes? I'd suggest using class="help-block" instead; then we could toggle all helptexts globally, at once

sbeaver-netgate committed:
<tt> => <pre>

Class changed from ‘notes’ to ‘help-block’ as suggested to accommodate
future global “verbose help” setting.
